app/models/donation_site.rb

```diff
@@ -40,10 +45,10 @@ def self.import_csv(csv, organization)
   end
 
   def self.csv_export_headers
-    %w{Name Address}
+    %w{Name Address Contact_Name Email Phone}
```

Comment:
CSV headers can have spaces so you are not forced to do something like `Contact_Name`. Looks like there is other places where they already use "Contact Name"  so that is a convention that should be respected.